.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Run CI on all branches, no need to restrict it since it is very quick and free 🤷‍♂️

Reply:
Done